docs/RoleReactions.md

```diff
@@ -0,0 +1,47 @@
+## Role Reactions
+
+Many servers choose to have roles that are assigned upon a user reacting to a message. You can set this up for your 
+server using Botomir.
+
+To do so, Botomir requires the following permissions:
```

Comment:
Those permissions need to be on the channel, which is different from the permissions that are set on the server, I'm not sure if that should be specified or not

Reply:
Ohhh, okay, lemme update that